INSTALLATION.md

```diff
@@ -0,0 +1,154 @@
+
+##### dnsmasq (default)
+
+> **_NOTE:_** If you have `NetworkManager` running, you must change its configuration to use `dnsmasq` as your local DNS. You can do this by running the following command:
+>
+>```sh
+>sudo cp /etc/NetworkManager/NetworkManager.conf /etc/NetworkManager/NetworkManager.conf_backup_thyra_install && sudo sed -i "s/keyfile/keyfile\ndns=dnsmasq/g" /etc/NetworkManager/NetworkManager.conf
+>```
+> This command backs up `/etc/NetworkManager/NetworkManager.conf` file to `/etc/NetworkManager/NetworkManager.conf_backup_thyra_install`.
+
+Then we must configure and restart the dnsmasq service:
+
+```sh
+sudo mkdir -p /etc/NetworkManager/dnsmasq.d/
+echo "address=/.massa/127.0.0.1" >> /etc/NetworkManager/dnsmasq.d/massa.conf
+sudo mv /etc/resolv.conf /etc/resolv.conf_backup_thyra_install && sudo ln -s /var/run/NetworkManager/resolv.conf /etc/resolv.conf
+sudo systemctl restart NetworkManager
+```
+
+> **_NOTE:_** your `/etc/resolv.conf` file has been backed up to `/etc/resolv.conf_backup_thyra_install`.
+
+Congratulations, you can now browse the **websites on-chain** seamlessly. If you need to take the pressure off, maybe a little [game](http://flappy.massa) can help.
+If you want to get down to business, you can start your [Massalian journey](http://my.massa) right away!
+
+### Windows
+
+#### Thyra installation
+
+Let's start by downloading the version of Thyra corresponding to your system [here](https://github.com/massalabs/thyra/releases/download/v0.0.2/thyra-server_windows_amd64).
+
+Next, you must rename it to `thyra-server.exe`.
+
+Congratulation, your version of Thyra is now installed on your system and can be run by executing `thyra-server`.
+
+> **_NOTE:_** If your DNS is already configured to handle the massa TLD, you're free to go. Otherwise, please follow the instructions in the next section.
+
+#### DNS installation
+
+> **_WARNING:_** If you already have a DNS service running that is not dnsmasq, you must configure it to redirect .massa to 127.0.0.1 (localhost).
+
+> **_PREREQUISITE:_** Have `Acrylic` already installed on your system. If not, you can follow the installation instructions [here](https://mayakron.altervista.org/support/acrylic/Home.htm) and the OS configuration [here](https://mayakron.altervista.org/support/acrylic/Windows10Configuration.htm).
+
+Let's start by configuring acrylic to redirect *.massa to locahost:
+1. Open Acrylic config file: Open Acrylic DNS Proxy UI > File > Open Acrylic Hosts
+
+2. Add `*.massa` top level domain to `AcrylicHosts.txt`: 
+```txt
+127.0.0.1   *.massa
+```
+3. Save the file and reload Acrylic: Open Acrylic DNS Proxy UI > Actions > Restart Acrylic Service
+
+Congratulations, you can now browse the **websites on-chain** seamlessly. If you need to take the pressure off, maybe a little [game](http://flappy.massa) can help.
+If you want to get down to business, you can start your [Massalian journey](http://my.massa) right away!
```

Comment:
If you want to get down to business, you can start your [Massalian journey](http://my.massa/thyra/wallet) right away!